lua/packer/log.lua

```diff
@@ -150,4 +150,10 @@ end
 log.new(default_config, true)
 -- }}}
 
+function log.open()
+  -- This should be consistent with the format mentioned in log.new above.
+  local logfile = string.format('%s/%s.log', vim.fn.stdpath('cache'), default_config.plugin)
+  vim.cmd(string.format("pedit %s", logfile))
```

Comment:
I wonder if preview is a good default for the log. As you mentioned in the issue this closes it seems like it will be quite hard to scan, but I haven't tried this locally, so I could be wrong.

Reply:
Yes, it would be. My suggestion would be to open it in a new tab and when the user will quit the tab, they will be redirected to the packer window. There's no additional code to be written here as with `tabedit`, the file will be opened in a new tab next to the current one and when that is closed, the user will be redirected to the previous tab.

Reply:
Tiny suggestion: `pedit`, `tabedit` and other similar commands support a `+cmd` argument that allows you to position the cursor at a specific location. `tabedit + foo` positions the cursor at the end of file `foo`, which can be pretty useful for logs

Reply:
TIL! I always wondered what `+cmd` do. Thanks for the suggestion! :)